commands/utils/SourceArchiveUtility.ts

```diff
@@ -73,42 +72,42 @@ export async function scheduleRunRequest(fileUri: vscode.Uri, requestType: runRe
     }
 
     //Schedule the run and Clean up.
-    status1.appendLine("Set up Run Request");
+    ext.outputChannel.appendLine("Set up Run Request");
```

Comment:
Sentence casing (i.e. only first word should be capitalized).  Title casing (all words capitalized) is generally only used by menus, buttons, titles etc.